src/behaviour.rs

```diff
@@ -1,14 +1,10 @@
 use bevy::prelude::*;
 
-#[cfg(feature = "inspect")]
-use bevy_inspector_egui::prelude::*;
 #[cfg(feature = "serialize")]
 use serde::{Deserialize, Serialize};
 
 #[derive(Reflect, Clone, Copy, Debug, Default, PartialEq, Eq)]
-#[cfg_attr(feature = "inspect", derive(InspectorOptions))]
```

Comment:
Maybe we could remove this part since, as you say, it now takes advantage of the reflect, but I don't think we should remove the feature

Reply:
But what's the point of the feature if it doesn't do anything?

Reply:
Avoid having too many dependencies when compiling in development and give the freedom to choose what to use. Think that compiling in Rust already takes too much time to have to compile things that I don't need, for example, not everyone needs to have the type inspection of this library and when they need it they just add the feature :D

Reply:
So it's a development-only feature? I still don't see for this would be useful for users of the crate, though...

Reply:
>  not everyone needs to have the type inspection of this library and when they need it they just add the feature :D

You understand that users of this crate won't get the dev-dependencies, even while they're developing, right?

Reply:
> You understand that users of this crate won't get the dev-dependencies, even while they're developing, right?

Yes, but precisely controlling the dependencies through the features allows them to choose in which environment they want to have it available, there will be those who want it in production, those who want it in development and those who do not want it

Reply:
But what does the feature do for them? They can't use it without also adding bevy-inspector-egui to their dependencies or dev-dependencies section? It does nothing except increase build times?